launcher/UpdateWindow.xaml.cs

```diff
@@ -28,6 +28,18 @@ public UpdateWindow()
 
         private async void Window_Loaded(object sender, RoutedEventArgs e)
         {
+        }
+        private void HandleProgress(object sender, DownloadProgressChangedEventArgs e)
+        {
+            UpdateBar.Value = e.ProgressPercentage;
+        }
+
+        private async void Confirm_Button_Click(object sender, RoutedEventArgs e)
+        {
+            Label Question0 = (Label)this.FindName("Question0");
```

Comment:
Couldn't you have just used Question0.Visibility etc?

Reply:
yes. intilizeComponent does create those properties already.

Reply:
both methods are fine but I think that this one just adds more clutter

Reply:
done